.github/workflows/backport.yml

```diff
@@ -0,0 +1,25 @@
+name: Backport
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    name: Backport
+    runs-on: ubuntu-latest
+    # Only react to merged PRs for security reasons.
+    # See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target.
+    if: >
+      github.event.pull_request.merged && (
+        github.event.action == 'closed'
+        || (
+          github.event.action == 'labeled'
+          && contains(github.event.label.name, 'backport')
```

Comment:
```suggestion
          && contains(github.event.label.name, 'needs backport')
```

Reply:
The github action's code force us to use this keyword. Also as it's done automatically we don't "need" to do anything 🎉